lib/components/term.js

```diff
@@ -299,11 +299,11 @@ export default class Term extends Component {
             height: '100%'
           }}
           /> :
-        <div
-          className={css('scrollbarShim')}
-          onMouseEnter={this.handleScrollEnter}
-          onMouseLeave={this.handleScrollLeave}
-          />
+            <div
```

Comment:
I had to indent the second part of the ternary here, not sure if this is intended behaviour or a bug. @sindresorhus?